containers/Makefile

```diff
@@ -203,8 +209,8 @@ build/signing-service/signing-service/RELEASE-VERSION: build/signing-service/sig
 build/signing-service/signing-service/: build/repositories/concent
 	cp --recursive "$</signing_service/." "$@"
 
-build/signing-service/middleman_protocol/: build/repositories/concent
-	cp --recursive "$</middleman_protocol/." "$@"
+# build/signing-service/middleman_protocol/: build/repositories/concent
+	# cp --recursive "$</middleman_protocol/." "$@"
```

Comment:
Why is this commented out?

If it's not needed, remove it.
If it's a workaround, there should be a comment explaining what you're working around.

Reply:
Remove